.travis.yml

```diff
@@ -23,7 +23,7 @@ matrix:
 
 install: composer update $COMPOSER_FLAGS -n
 
-script: vendor/bin/phpunit -v --coverage-clover=coverage.clover
+script: vendor/bin/phpunit --verbose --coverage-clover=coverage.clover
```

Comment:
@stof Sometimes `-v` is short for `--version` (e.g. `php -v`), that's why I'd like to be explicit here.